component-builder/setup.py

```diff
@@ -37,7 +37,7 @@
     install_requires=[
         'bash>=0.5',
         'docopt',
-        'github3.py>=1.0.0a3',
+        'github3.py>=1.0.0',
```

Comment:
Should we be more cautious with versions, to prevent these kinds of incompatible situations? E.g. something like:
```suggestion
        'github3.py~=1.0.0',
```

Reply:
Yes, this one is better.